### ISSUE TITLE
Dot in suffix 

### ISSUE THREAD

**Issue by nmartinezb3** [s-yadav/react-number-format#14]
Hi,

I'm trying to set a suffix with a ".", and it is not working:
```

<NumberFormat 
      thousandSeparator={true} 
      prefix={'$'}
      suffix={' per sq. ft.'}
      placeholder={this.props.placeholder} 
      value={this.props.value} 
      onChange={(event, value) => console.log(value)} />
```
When I input something, nothing is displayed in the input and the console throws:
```
The specified value "NaN" is not a valid number. The value must match to the following regular expression: -?(\d+|\d+\.\d+|\.\d+)([eE][-+]?\d+)?
```
If I remove the dots from the suffix (per sq ft) it works ok.

**Comment by s-yadav** [comment]
I will fix this.

**Comment by s-yadav** [comment]
Fixed on 2.0.0 beta
